doc/source/whatsnew/v1.1.0.rst

```diff
@@ -902,6 +767,94 @@ Interval
 
 Indexing
 ^^^^^^^^
+
```

Comment:
hmm you really want this in the middle of the bug fix section?

Reply:
I can shorten these to one-sentence descriptions and ensure that the linked issue has a clear example. Would that be preferable?